_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -1232,15 +1232,24 @@ public static function rewind_data() {
 			return new WP_Error( 'site_id_missing' );
 		}
 
+		if ( ! isset( $_GET['_cacheBuster'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
I feel like this could expose us to a (reflected?) DoS attack. The rewind state endpoint can be a heavy/long operation, if this request is made multiple times, it could leave the requesting site in a race condition.

Reply:
Good point. This could potentially happen if the request are done by a logging user that is making the request with the correct nonce. 

If those things are not in place then the request will not reach .com servers. 
So I think we are good at this moment.